components/src/preact/mutations/mutations.tsx

```diff
@@ -207,7 +207,30 @@ const Toolbar: FunctionComponent<ToolbarProps> = ({
                     filename='insertions.csv'
                 />
             )}
-            <Info>Info for mutations</Info>
+            <MutationsInfo />
         </>
     );
 };
+
+const MutationsInfo = () => (
+    <Info>
+        <InfoHeadline1>Mutations</InfoHeadline1>
+        <InfoParagraph>
+            This shows mutations of a variant. There are three types of mutations:{' '}
+            <InfoLink href='https://www.genome.gov/genetics-glossary/Substitution'>substitutions</InfoLink>,{' '}
+            <InfoLink href='https://www.genome.gov/genetics-glossary/Deletion'>deletions</InfoLink> and{' '}
+            <InfoLink href='https://www.genome.gov/genetics-glossary/Insertion'>insertions</InfoLink>.
+        </InfoParagraph>
+        <InfoHeadline2>Proportion calculation</InfoHeadline2>
+        <InfoParagraph>
+            The proportion of a mutation is calculated by dividing the number of sequences with the mutation by the
+            total number of sequences with a non-ambiguous symbol at the position.
+        </InfoParagraph>
+        <InfoParagraph>
+            <b>Example:</b> If we look at nucleotide mutations at position 5 where the reference has a T: if there are
```

Comment:
```suggestion
            <b>Example:</b> Assume we look at nucleotide mutations at position 5 where the reference has a T and there are
```
Two chained ifs. And I would put some of the text in bullet points like this:

Assume we look at nucleotide mutations at position 5 where the reference has a T and there are 10 sequences in total:

- 3 sequences have a C
- 2 sequences have a T
- 1 sequence has a G
- 3 sequences have an N
- 1 sequence has a Y (which means T or C)
 
then the proportion ...

Reply:
I would also add a link to the ambiguous symbols.

Reply:
Which link? Is it important to have this link here? The ambiguous symbols don't show up in the table/grid, do they?

Reply:
I dont know which link. From a short google search I came up with: https://www.dnabaser.com/articles/IUPAC%20ambiguity%20codes.html (Dont know if this is a good way to present this). I just thought it would be nice to have the reference here. Or a link to our doku page. 
But your right, they dont show in the grid.